crank/crankServerAPI.py

```diff
@@ -1,33 +0,0 @@
-# crankServerAPI.py
```

Comment:
I originally separated the ServerAPI with crankAPI, because only the functions defined in serverAPI module need to be loaded and run by the server, while the functions in crankAPI are expected to be running on a "computing node" or "container"

Reply:
You missed the get next jobs function I believe, either way it would be good to have the entire API in one place to make things a bit simpler IMO.

Reply:
The `crankAPI.get_next_jobs()` function is part of the crankAPI, which does a lot of computation and I think should run in the "crank computing node" or "crank container". The functions I put in crankServerAPI are the ones that need to run on the server, and I made them as light-weight as possible. Maybe I had a different understanding of how the workflow runs.